### ISSUE TITLE
Add option to set control limits

### ISSUE THREAD

**Issue by simonthelwall** [nhs-r-community/FunnelPlotR#44]
At the moment, the control limits are fixed at 95 and 99.8%. This may not be standard for all applications of funnel plots. It would be nice to have options to set these in `funnel_plot`.

**Comment by chrismainey** [comment]
Yes, that would be a really nice feature.  It need some substantial development time to redo this.  The 95 and 99.8 correspond to the 2 and 3 standard error (sigma) type limits, but from a probability density perspective, they are 95% and 99.8%.  Would be very happy to receive pull requests or have collaborators to rebuild it though, if you are interested?
